polls/views.py

```diff
@@ -37,6 +37,10 @@ class ResultsView(generic.DetailView):
     model = Question
     template_name = 'polls/results.html'
 
+# !!! EDITED !!!
```

Comment:
Forgot to take this edit tag out. I usually indicate things I edit with !!! to get my attention.